snow/validators/set.go

```diff
@@ -318,6 +237,13 @@ func (s *vdrSet) len() int {
 	return len(s.vdrSlice)
 }
 
+func (s *vdrSet) HasCallbackRegistered() bool {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
+
+	return len(s.callbackListeners) > 0
+}
```

Comment:
Does this need to be exported? I feel it can be private.

Reply:
unexported it